src/prefect/cli/task.py

```diff
@@ -1,26 +1,51 @@
-from typing import TYPE_CHECKING, Any
+from __future__ import annotations
+
+import inspect
+import logging
+from typing import Any, Optional
 
 import typer
 
 from prefect.cli._types import PrefectTyper
 from prefect.cli._utilities import exit_with_error
 from prefect.cli.root import app
+from prefect.logging import get_logger
 from prefect.task_worker import serve as task_serve
-from prefect.utilities.importtools import import_object
-
-if TYPE_CHECKING:
-    from prefect.tasks import Task
+from prefect.tasks import Task
+from prefect.utilities.importtools import import_object, load_module
 
 task_app: PrefectTyper = PrefectTyper(name="task", help="Work with task scheduling.")
 app.add_typer(task_app, aliases=["task"])
 
+logger: logging.Logger = get_logger("prefect.cli.task")
+
+
+def _import_tasks_from_module(module: str) -> list[Task[Any, Any]]:
+    try:
+        mod = load_module(module)
+    except ModuleNotFoundError:
+        exit_with_error(
+            f"Module '{module}' could not be imported. Please check the module name and try again."
+        )
```

Comment:
I think it makes more sense to have this error handling in the command function so that this function can be agnostic to CLI stuff.

Reply:
good point